### ISSUE TITLE
NPC profile breaks on refresh

### ISSUE THREAD

**Issue by EirikOsnes** [ryceg/Eigengrau-s-Essential-Establishment-Generator#280]
![image](https://user-images.githubusercontent.com/25607450/84056051-f9012d00-a9b5-11ea-905e-20a26ac5a757.png)

For context, the stacktraces for each of the errors from top down:
```
<<run setup.expandNPC($town, $npcs[$selected.key])>>
```

```
<<if setup.socialClass[$currentNPC.socialClass].lifestyle.includes(_lifestyleStandard[1])>>
  \<<print $currentNPC.heshe.toUpperFirst()>> belongs to the <<print setup.createTippyFull($currentNPC.firstName + " " + _lifestyleStandard[3], $currentNPC.socialClass)>> social class
\<<else>>
  \<<print $currentNPC.heshe.toUpperFirst()>> belongs to the $currentNPC.socialClass social class, but _lifestyleStandard[3]
\<</if>>
```

8 identical instances of this:
```
<<set _finances[_key] to _value($town, $currentNPC)>>
```

```
<<set _finances.tax to _finances.grossIncome - _finances.netIncome, _finances.totalLifestyleExpenses to _finances.lifestyleStandard[0] + _finances.lifestyleExpenses>>
```

Using Chrome,
https://eigengrausgenerator.com/#nippyunfoldedcarpenterant

**Comment by Glinkis** [comment]
I think this happens only after refreshing the page?

**Comment by EirikOsnes** [comment]
Not to my knowledge - opened another NPC that I for sure had not opened before, and got the same.

**Comment by ryceg** [comment]
Can you please provide steps to reach Aelar Cithreth?

**Comment by ryceg** [comment]
Given that Aelar is not generated on load, it is difficult to reproduce this in absence of steps. That said, the above bug _can_ be reproduced by refreshing the page while on an NPC profile. Therefore, I am renaming this issue to reflect it occuring on refresh.

**Comment by ryceg** [comment]
Closed as #325 has more information
